README.rst

```diff
@@ -237,14 +233,22 @@ decorator::
 Tests and local development
 ---------------------------
 
-To run the tests, docker is used::
+(Re)create & activate a virtualenv::
+
+    $ rm -rf .venv
+    $ virtualenv .venv --python=python3
+    $ source .venv/bin/activate
 
-    $ docker-compose build
-    $ docker-compose run --rm web buildout
-    $ docker-compose run web bin/test
+Install package and run tests::
+
+    (virtualenv)$ pip install django==2.2
+    (virtualenv)$ pip install -e .[test]
+    (virtualenv)$ python manage.py test
```

Comment:
Zo'n virtualenv activeren is wel standaard.

Op zich werkt alles ook als je `bin/python manage.py` doet enzo.
- Werkt ook als je de virtuenv geactiveerd hebt.
- Scheelt die `(virtualenv)` voor de dollartekens. Die `(virtualenv)` zou ik toch weglaten.
- Die heb je toch niet nodig als je `bin/pytest` doet.

Reply:
Ik volg je hier niet helemaal. Wel/niet de virtualenv activeren? En bedoel je `.venv/bin/python` en `.venv/bin/pytest`?

Reply:
Ah ik geloof dat ik het begrijp. Is prima, ik zal het wijzigen